### ISSUE TITLE
提供的模型网络结构

### ISSUE THREAD

**Issue by Wazxcvb** [Meizhen2023/SARID#4]
您好，麻烦您能否提供一下logs文件夹下模型的网络结构，该模型与现有代码中的transformer的网络结构对不上

**Comment by Meizhen2023** [comment]
Yes please check the file "model_structure_for_[model_epoch_best_R.ckpt].py" in logs. (This is only for Transformer-based model, and the feature is MFCC, with n_mfcc equals 128)
